app/views/pages/_header.html.erb

```diff
@@ -1,28 +1,42 @@
 <nav class="navbar navbar-expand-lg navbar-light bg-light">
   <div class="container-fluid">
-   <%= link_to "Firma", root_path, class:"navbar-brand" %>
-    
+
+   <%= link_to image_tag("https://media1.thehungryjpeg.com/thumbs2/ori_3490122_b6f8966043e8b6950749930f8a2c9475ee03de54_happy-smile-smiling-face-on-yellow-background-laughter-logo-funny-v.jpg", size:"70x45"), root_path, class:"navbar-brand" %>
+  
     <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
       <span class="navbar-toggler-icon"></span>
     </button>
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
       <ul class="navbar-nav me-auto mb-2 mb-lg-0">
 
-      <% if employee_signed_in? %>
-          <li class="nav-item">
-          <%= link_to "Log Out", destroy_employee_session_path, method: :delete, class: "nav-link" %>
-          </li>
-
-      <% else %>
+      <% if admin_user_signed_in? %>
         <li class="nav-item">
-          <%= link_to "Sign Up", new_employee_registration_path, class: "nav-link" %>
-        </li>
-        
-        <li class="nav-item">
-          <%= link_to "Log In", new_employee_session_path, class: "nav-link" %>
+          <%= link_to "Log Out", destroy_admin_user_session_path, method: :delete, class: "nav-link" %>
+      
+      <% else %>
+
+          <% if employee_signed_in? %>
+            <li class="nav-item">
+              <%= link_to "Log Out", destroy_employee_session_path, method: :delete, class: "nav-link" %>
+            </li>
+
+          <% else %>
+            <li class="nav-item">
+              <%= link_to "Sign Up", new_employee_registration_path, class: "nav-link" %>
+            </li>
+            
+            <li class="nav-item">
+              <%= link_to "Log In", new_employee_session_path, class: "nav-link" %>
+            </li>
+
+            <li class="nav-item">
+            <%= link_to "Admin Log In", admin_root_path, class:"nav-link", style: "color:green"%>
         </li>
+          <% end %>
+
       <% end %>
```

Comment:
When you create separate header for admin, then there will be no need to use that many `if`s

Reply:
I added admin header :)